main.go

```diff
@@ -30,6 +30,7 @@ const (
 	ENV_WG_TUN_FD             = "WG_TUN_FD"
 	ENV_WG_UAPI_FD            = "WG_UAPI_FD"
 	ENV_WG_PROCESS_FOREGROUND = "WG_PROCESS_FOREGROUND"
+	ENV_WG_TUN_NAME           = "WG_TUN_NAME"
```

Comment:
This is part of my cheating to avoid having to do complex stuff to identify which tun PPA we are using by examining the file descriptor. I've added some additional comments in the tun_illumos file to explain.
I don't know if @zx2c4 will let me get away with this cheating or not... :)